pipelines/nextflow/workflows/dumper_pipeline/main.nf

```diff
@@ -103,21 +105,33 @@ if (params.host && params.port && params.user && params.output_dir) {
     exit 1, "Missing server parameters"
 }
 
+// Select the files to dump
+dump_meta = default_selection
+dump_sql = true
+dump_number = 8
 if (params.select_dump) {
-    items = params.select_dump.split(/,/)
-    for (item in items) {
+    dump_meta = []
+    dump_sql = false
+    dump_number = 0
+    dump_meta = params.select_dump.split(/,/).collect().unique()
```

Comment:
This is clearly getting to the point where we need a function that can split input arguments into a list, and even accept a list of "choices" so the code below that raises exceptions is handled by the "parser".